src/pages/documentation/mobile-core/lifecycle/tabs/api-reference.md

```diff
@@ -8,13 +8,29 @@ final String lifecycleExtensionVersion = Lifecycle.extensionVersion();
 
 <Variant platform="ios" api="extension-version" repeat="4"/>
```

Comment:
Did we add any examples here? Or are we just changing the body of the content?

If we added paragraphs, we're going to need to add to the "repeat" value, since it's not more paragraphs (i'm aware it's a dumb system, but unfortunately it's what we have.)

So, in very plain terms, if we added the syntax/examples for all of them if they only had examples before, we're going to need to increase the repeat amount.

Reply:
I don't understand the system sorry. What does repeat do? 

There are three APIs i changed here. 

1. extensionVersion: It had examples, but it didn't have a syntax. And it didn't say "Example" above the examples.
2. lifecycleStart: It had an example, but i added another example (needs two to show how to use the API in two ways), and it was missing syntax.
3. lifecyclePause: Same as 1.

Reply:
Got it!

Repeat basically just tells the VariantBlock how many paragraphs exist within the block. So, in this case, with the Syntax + Examples added, it'll be like 10, since there are 10 blocks within the Variant. I hope that makes sense?

Reply:
Got it. Thanks for the explanation. Let me know if that looks good now.